OpenSSL on Mac does not build

If one removes openssl from brew on Mac, he/she gets : 
```DEBUG:flpproto:OpenSSL:0: Configured for darwin-i386-cc.
DEBUG:flpproto:OpenSSL:0: 
DEBUG:flpproto:OpenSSL:0: *** Because of configuration changes, you MUST do the following before
DEBUG:flpproto:OpenSSL:0: *** building:
DEBUG:flpproto:OpenSSL:0: 
DEBUG:flpproto:OpenSSL:0: 	make depend
DEBUG:flpproto:OpenSSL:0: + make depend
DEBUG:flpproto:OpenSSL:0: making depend in crypto...
DEBUG:flpproto:OpenSSL:0: clang: error: argument unused during compilation: '-Wa,--noexecstack' [-Werror,-Wunused-command-line-argument]
DEBUG:flpproto:OpenSSL:0: make[1]: *** [local_depend] Error 1
DEBUG:flpproto:OpenSSL:0: make: *** [depend] Error 1
DEBUG:flpproto:OpenSSL:0: 
ERROR:flpproto:OpenSSL:0: Error while executing /Users/aldaqci/alice-jenkins-recipes/sw/SPECS/osx_x86-64/OpenSSL/v1.0.2o-1/build.sh on ```

--- a/defaults-o2-dataflow.sh
+++ b/defaults-o2-dataflow.sh
@@ -48,7 +48,7 @@ overrides:
       - FreeType:(?!osx)
       - Python-modules
       - libxml2
-      - OpenSSL
+      - OpenSSL:(?!osx)
       - "GCC-Toolchain:(?!osx)"
       - libpng
       - lzma

--- a/defaults-o2-dev-fairroot.sh
+++ b/defaults-o2-dev-fairroot.sh
@@ -39,7 +39,7 @@ overrides:
       - FreeType:(?!osx)
       - Python-modules
       - libxml2
-      - OpenSSL
+      - OpenSSL:(?!osx)
       - "GCC-Toolchain:(?!osx)"
       - libpng
       - lzma

